rpc/wrtc_call_queue_mongodb.go

```diff
@@ -607,10 +613,11 @@ func (queue *mongoDBWebRTCCallQueue) processNextSubscriptionEvent(next mongoutil
 				// though, we want to send the events as fast as possible as mentioned
 				// above and cannot block on the send to see if the receiver locked
 				// the document.
-				if answerChan.Send(event) {
+				if answerChan.Send(event, queue.logger) {
 					return
 				}
 			}
+			queue.logger.Infof("Number of answer channels: %d", len(answerChans))
```

Comment:
Necessary? Looks like this is logged in the warn below.

Reply:
Oops just saw your reference to this in the PR description, could you move this above the for loop maybe? If I understand correctly, the happy path would not get to this point (we'd `return` on L616 since we were able to send to some `answerChan`.

Reply:
Oops you're right, will move then merge!